cmd/metal-api/internal/grpc/supwdServer.go

```diff
@@ -0,0 +1,22 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	v1 "github.com/metal-stack/metal-api/pkg/api/v1"
+	"io/ioutil"
+	"strings"
+)
+
+func (s *Server) FetchSupermetalPassword(ctx context.Context, req *v1.SupermetalPasswordRequest) (*v1.SupermetalPasswordResponse, error) {
```

Comment:
`Supermetal` or `SuperMetal`? Or maybe `SuperPassword` is already enough? Why do we need `Metal` in the name after all?

Reply:
At least we need some context, since SuperPassword does not say anything regarding the corresponding user, which is the bmc/ipmi superuser (https://github.com/metal-stack/metal-hammer/issues/25).

Since our default bmc/ipmi username is `metal` I chose the name `supermetal` to indicate this fact.